analyzer/src/funTest/assets/projects/synthetic/bower-expected-output.yml

```diff
@@ -0,0 +1,160 @@
+---
+project:
+  id: "Bower::a-test-project:"
+  definition_file_path: "<REPLACE_DEFINITION_FILE_PATH>"
+  declared_licenses:
+  - "MIT"
+  vcs:
+    type: ""
+    url: ""
+    revision: ""
+    path: ""
+  vcs_processed:
+    type: "git"
+    url: "<REPLACE_URL>"
+    revision: "<REPLACE_REVISION>"
+    path: "<REPLACE_PATH>"
+  homepage_url: "https://github.com/heremaps/oss-review-toolkit"
+  scopes:
+  - name: "dependencies"
+    dependencies:
+    - id: "Bower::polymer:2.4.0"
+      dependencies:
+      - id: "Bower::shadycss:1.5.0"
+        dependencies: []
+        errors: []
+      - id: "Bower::webcomponentsjs:1.2.6"
+        dependencies: []
+        errors: []
+      errors: []
+  - name: "devDependencies"
+    dependencies:
+    - id: "Bower::jquery:3.3.1"
+      dependencies: []
+      errors: []
+packages:
+- package:
+    id: "Bower::a-test-project:"
+    declared_licenses:
+    - "MIT"
+    description: "This string describes the test project."
+    homepage_url: "https://github.com/heremaps/oss-review-toolkit"
+    binary_artifact:
+      url: ""
+      hash: ""
+      hash_algorithm: ""
+    source_artifact:
+      url: ""
+      hash: ""
+      hash_algorithm: ""
+    vcs:
+      type: ""
+      url: ""
+      revision: ""
+      path: ""
+    vcs_processed:
+      type: ""
+      url: ""
+      revision: ""
+      path: ""
+  curations: []
```

Comment:
The `Project` itself should not appear in the package list.

Reply:
Done.